scripts/suite_objects.py

```diff
@@ -143,6 +143,11 @@ def call_string(self, cldicts=None, is_func_call=False, subname=None):
                         raise CCPPError(errmsg.format(stdname, clnames))
                     # end if
                     lname = dvar.get_prop_value('local_name')
+                    # Optional variables in the caps are associated with 
+                    # local pointers of <lname>_ptr
+                    if dvar.get_prop_value('optional'):
+                        lname = dummy+'_ptr'
```

Comment:
is there a way to save this local pointer name in an object somewhere? Not critical, but we're manually adding "_ptr" both here and on line 1243, which I imagine could cause someone grief if they ever wanted to change the name of the variable.

Reply:
This will be addressed in the documentation.
The idea is that each optional scheme argument will have a temporary pointer in the cap with the _ptr suffix, which shouldn't interfere with the local variable name. We don't query this variable by name, we assume that it exists when there is an optional variable.